### PR TITLE
feat: host Inter font (#1981)

### PR DESCRIPTION
**Pull request by andre-code**
This RP is to host the font "Inter".

/deploy #persist

### CODE REVIEW

File: client/src/styles/fonts/_fonts.scss [hunk 1]
```diff
@@ -3,8 +3,21 @@
...
 */
 
-/* Webfont: Calcutta-Bold */
-@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;600&display=swap');
+@font-face {
+  font-family: 'Inter';
+  font-style: normal;
+  font-weight: 400;
+  src: local('Inter Regular'), local('Inter-Regular'),
+  url('../fonts/Inter/static/Inter-Regular.ttf') format('ttf');
```
Comment (ciyer) [inline client/src/styles/fonts/_fonts.scss:11]: I suspect this path is not correct.

**Review by ciyer — CHANGES_REQUESTED** [review]
I think the path to the font is not correct. This is what it looks like on my Mac

image

That is in **Helvetica**.

It should like like this:

image

(E.g., look at the dot on the _lowercase-i_. It is square in Helvetica, round in Inter).

File: client/src/styles/fonts/_fonts.scss (hunk 1 above)
Comment (andre-code) [reply client/src/styles/fonts/_fonts.scss]: yes, was missing `font-display: swap;`

Should work now in all browsers

**Review by ciyer — APPROVED** [review]
👍

**Pull request merged**